lib/web_ui/lib/src/engine/html/scene.dart

```diff
@@ -39,13 +44,13 @@ class PersistedScene extends PersistedContainerSurface {
     //                update this code when we add add2app support.
     final double screenWidth = html.window.innerWidth!.toDouble();
     final double screenHeight = html.window.innerHeight!.toDouble();
-    _localClipBounds = ui.Rect.fromLTRB(0, 0, screenWidth, screenHeight);
-    _localTransformInverse = Matrix4.identity();
-    _projectedClip = null;
+    localClipBounds = ui.Rect.fromLTRB(0, 0, screenWidth, screenHeight);
+    localTransformInverse = Matrix4.identity();
+    projectedClip = null;
   }
 
   @override
-  Matrix4? get localTransformInverse => _localTransformInverse;
+  Matrix4? get defaultLocalTransformInverse => null;
```

Comment:
Comment suggestion: Lazily constructs or returns cached copy of inverse of local transforms.

Reply:
maybe name this effectiveLocalTransformInverse instead of default prefix? . Other option would be to rename Matrix4 localTransformInverse to cachedLocalTransformInverse and not rename this one.